projects/ideabox-groups.md

```diff
@@ -0,0 +1,210 @@
+
+
+Color, font, and icon spec:
+
+![color font and icon specs][button-hover-specs]
+
+#### Link to icon files
+
+You will need the `svg` files for the delete, upvote, and downvote icons. [Here's the link.](https://drive.google.com/open?id=0B_lPnjyMN6-CaHpTQlRUdzNnZ0U)
+
+[desktop-base]: /assets/images/projects/ideabox/ideabox-01.png
+[mobile-base]: /assets/images/projects/ideabox/ideabox-02.png
+[mobile-full]: /assets/images/projects/ideabox/ideabox-03.png
+[button-hover-specs]: /assets/images/projects/ideabox/ideabox-04.png
+
+## Rubric
+
+### Functional Expectations
+
+* [ ]  Novice - Application meets all of the basic functional expectations of create, edit, delete, and those changes persist in `localStorage`
+* [ ]  Advanced Beginner - Application adds 'Changing the quality of an idea' and enables 'Filtering and Searching by Text' as defined in the spec
+* [ ]  Proficient - Applications adds 'Filter by Importance' and 'Recent' Idea as outlined in the spec
+* [ ]  Exceptional - Application adds at least one of the extensions
+
+------------------------------------------------------------------
+
+### Comp Recreation
+
+* [ ]  Novice - Application implements all major comp details correctly (colors, fonts, icons, etc.)
+* [ ]  Advanced Beginner - Application adds adequate sizing and spacing of elements on at least 1 screen size
+* [ ]  Proficient - Application adds adequate sizing and spacing of elements at all screen sizes
+* [ ]  Exceptional - Application adds exceptional sizing and spacing of elements at all screen sizes
+
+------------------------------------------------------------------
+
+### HTML - Style and Implementation
+
+* [ ]  Novice - Crafts markup according to the [turing html style guide](https://github.com/turingschool-examples/html)
+* [ ]  Advanced Beginner - Application adds to the above by using data-* attributes for all data related things
+* [ ]  Proficient - Applications adds to the above with markup that is easy to read and follow across naming conventions
+* [ ]  Exceptional - Application adds to the above by using BEM, SMACCS, or another set of naming conventions for classes
+
+------------------------------------------------------------------
+
+### CSS - Style and Implementation
+
+* [ ]  Novice - Crafts CSS according to the [turing css style guide](https://github.com/turingschool-examples/css)
+* [ ]  Advanced Beginner - Application adds organization for the whole stylesheet and within rules
+* [ ]  Proficient - Applications adds to the above by removing repetitive rules and blocks of code according to the DRY principle 
+* [ ]  Exceptional - Application adds to the above by using BEM, SMACCS, or another set of naming conventions for classes
+
+------------------------------------------------------------------
+
+### JAVASCRIPT - Implementation and style
+
+* [ ]  Novice - Crafts JS according to the [turing js style guide](https://github.com/turingschool-examples/javascript/tree/master/es5)
```

Comment:
Cheers to including the style guides in the rubric.  I really like this.  Rubric is really clear in it's goals.  Awesome!